src/InfrastructureSystems.jl

```diff
@@ -2,6 +2,10 @@ isdefined(Base, :__precompile__) && __precompile__()
 
 module InfrastructureSystems
 
+# Cost aliases don't display properly unless they are exported from IS
```

Comment:
@jd-lara you won't love this, but exporting the aliases is the [only way](https://github.com/JuliaLang/julia/blob/master/base/show.jl#L617) way to get them to register as aliases for the purposes of printing and such, and I think overriding the printing of these types is uglier than adding them as exports. @daniel-thom agrees that this is fairly harmless — a name collision is unlikely because these would be imported rather than redefined in any package that also exports, and also we are (possibly accidentally) [already exporting](https://github.com/NREL-Sienna/InfrastructureSystems.jl/issues/391) some things from IS.

Reply:
uggg. Fine